lib/scss/layouts/stanford-page.layout.scss

```diff
@@ -0,0 +1,8 @@
+@charset 'UTF-8';
+@import '../jumpstart_ui.config';
+
+.node-stanford-page-su-page-components {
+  @media #{$su-breakpoint-xl} {
+    padding-left: 150px;
```

Comment:
This should use the `@import padding()` mixin and the `$su-screen-margins` variable map.

Reply:
The $su-screen-margins doesn't have a size that is equivalent to 150px.
https://decanter.stanford.edu/docs/scss/index.html#variable-$su-screen-margins

Reply:
Ah, wrong gutter. My bad.